packages/driver/cypress/integration/e2e/multi-domain/commands/multi_domain_screenshot.spec.ts

```diff
@@ -1,262 +1,266 @@
 // @ts-ignore / session support is needed for visiting about:blank between tests
 context('multi-domain screenshot', { experimentalSessionSupport: true }, () => {
-  beforeEach(() => {
-    this.serverResult = {
-      path: '/path/to/screenshot',
-      size: 12,
-      dimensions: { width: 20, height: 20 },
-      multipart: false,
-      pixelRatio: 1,
-      takenAt: new Date().toISOString(),
-      name: 'name',
-      blackout: [],
-      testAttemptIndex: 0,
-      duration: 100,
-    }
-
-    cy.visit('/fixtures/multi-domain.html')
-    cy.get('a[data-cy="screenshots-link"]').click()
-  })
-
-  it('captures the fullPage', () => {
```

Comment:
Why was the fullPage test removed?

Reply:
Good catch!, It was a [bad merge](https://github.com/cypress-io/cypress/pull/20561/commits/2e3b1851fd5aac5ba6788b94656dd3bb0c355872#diff-fe7e0c4706ee3ce6b8c6e47c1e0e5cdd22843893d31bc53ff7cbaed88e497eebL24-L37). I've restored it